node/src/store/indexed_db_store.rs

```diff
@@ -222,7 +222,7 @@ impl IndexedDbStore {
             return false;
         };
 
-        height <= head_height
+        height > 0 && height <= head_height
```

Comment:
here height also can't be zero. It would mean someone put a header with height zero but it would fail on validation

Reply:
ignore me :sweat_smile: